conans/model/manifest.py

```diff
@@ -36,9 +37,10 @@ def gather_files(folder):
             if os.path.exists(abs_path):
                 file_dict[rel_path] = abs_path
             else:
-                raise ConanException("The file is a broken symlink, verify that "
-                                     "you are packaging the needed destination files: '%s'"
-                                     % abs_path)
+                if not get_env("CONAN_SKIP_BROKEN_SYMLINKS_CHECK", False):
```

Comment:
What if we completely remove this check? Would it be breaking behavior?

Reply:
I don't think it would be breaking behavior. The question is if this check is avoiding some headaches when the broken symlinks are not intended or not. To be honest, I don't know...